### PR TITLE
add_test_cases

### PR DESCRIPTION
**Pull request by jenniexie**

### CODE REVIEW

**Review by qilinkang — CHANGES_REQUESTED** [review]
. No cambrionix module under src.tools:
from src.tools import cambrionix

. Should we use a constant variable to define the timeout instead of 20 in the following line ? 20s maybe not enough for a device to join network and become a router.

return verify_within(check_device_role, 20, delay_time=5)

. We can use this constant for assert lines like this one:
self.assertTrue(r1_result, 'r1 cannot get into leader role after 180 seconds timeout')

**Comment by jenniexie** [comment]
> . No cambrionix module under src.tools:
> from src.tools import cambrionix
> 
> . Should we use a constant variable to define the timeout instead of 20 in the following line ? 20s maybe not enough for a device to join network and become a router.
> 
> return verify_within(check_device_role, 20, delay_time=5)
> 
> . We can use this constant for assert lines like this one:
> self.assertTrue(r1_result, 'r1 cannot get into leader role after 180 seconds timeout')

I have fixed all comments

**Review by qilinkang — APPROVED** [review]
LGTM

File: src/tests/openthread/ot_test_leader_loss.py [hunk 1]
```diff
@@ -0,0 +1,268 @@
+from src.config import wpan_constants as wpan
```
Comment (jwhui) [inline src/tests/openthread/ot_test_leader_loss.py:15]: License header?

File: src/tests/openthread/ot_test_partition_merge.py [hunk 2]
```diff
@@ -0,0 +1,230 @@
+from src.config import wpan_constants as wpan
```
Comment (jwhui) [inline src/tests/openthread/ot_test_partition_merge.py:15]: License header?

Comment (jenniexie) [reply src/tests/openthread/ot_test_partition_merge.py:15]: fixed

File: src/tests/openthread/ot_test_leader_loss.py (hunk 1 above)
Comment (jenniexie) [reply src/tests/openthread/ot_test_leader_loss.py:15]: fixed

**Review by jwhui — APPROVED** [review]

**Pull request merged**